### PR TITLE
fix(ds): Panels are cropped-out

### PR DESCRIPTION
**Pull request by goombeer**
# Background

https://tailor.atlassian.net/browse/FL-166

# Changes

When data is available

https://github.com/tailor-platform/frontend-packages/assets/28993018/2e30d0e6-51b1-4d51-8aae-d05723630843

When there is no data

https://github.com/tailor-platform/frontend-packages/assets/28993018/a5d59d53-d8f9-44c0-9c25-570a947e8cc4

### CODE REVIEW

File: packages/design-systems/src/components/composite/Datagrid/ColumnFeature/PinnedColumn.tsx [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
 import { Box } from "@tailor-platform/styled-system/jsx";
 import type { Column } from "@tanstack/react-table";
 import { css } from "@tailor-platform/styled-system/css";
 import { MoreVertical as MoreVerticalIcon } from "lucide-react";
+import { createPortal } from "react-dom";
 import { Button } from "../../../Button";
 import { addEventOutside } from "../addEventOutside";
 import type { Localization } from "@locales/types";
 
+const ModalPortal = ({ children }: { children: React.ReactNode }) => {
```
Comment (goombeer) [inline packages/design-systems/src/components/composite/Datagrid/ColumnFeature/PinnedColumn.tsx:11]: I thought about using Portal from arc-UI, but when I tried to use it, the display flickered, so I decided to use this implementation.
ref) https://github.com/chakra-ui/ark/blob/e566a03c1d202b2a6d193da8e9b4fa69d4428805/frameworks/react/src/components/portal/portal.tsx#L22

File: packages/design-systems/tsup.config.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ const devOpts: Options =
 
 const commonOpts: Options = {
   format: ["cjs", "esm"],
-  external: ["react"],
+  external: ["react", "react-dom"],
```
Comment (goombeer) [inline packages/design-systems/tsup.config.ts:15]: Fixed for display in Storybook

**Review by IzumiSy — COMMENTED** [review]
Bump a minor version up on package.json please

**Review by murayama-r — APPROVED** [review]
LGTM

**Review by IzumiSy — APPROVED** [review]

**Pull request merged**